### PR TITLE
ACS-528 Fix alfresco bugs, add roles pki-aws* conditional

### PR DESCRIPTION
**Pull request by martinbozinovski**
Fix pip, alfresco and add conditions for pki-aws-* roles

### CODE REVIEW

**Review by dzdimov — APPROVED** [review]

File: vagrant/provisioning/arkcase-ce.yml [hunk 1]
```diff
@@ -5,8 +5,10 @@
       tags: [core, common]
     - role: pki-aws-csr
       tags: [pki-aws-csr]
+      when: aws_client is undefined or aws_client
```
Comment (bojmil-armedia) [inline vagrant/provisioning/arkcase-ce.yml:8]: aws_client | default(true)

File: vagrant/provisioning/roles/arkcase-developer/tasks/templates/start_services.sh [hunk 2]
```diff
@@ -3,7 +3,7 @@
 ##if the services are not running it will start the services
 
 ##list the services to check
-SERVICES=('haproxy' 'config-server' 'mariadb' 'httpd' 'pentaho' 'snowbound' 'solr' 'alfresco' 'activemq' 'confluent-kafka' 'confluent-zookeeper' 'confluent-control-center' 'confluent-kafka-connect' 'confluent-kafka-rest' 'confluent-ksql' 'confluent-kafka-connect' 'confluent-schema-registry' 'mongod' 'samba')
+SERVICES=('haproxy' 'config-server' 'mariadb' 'httpd' 'pentaho' 'snowbound' 'solr' 'alfresco' 'activemq' 'confluent-kafka' 'confluent-zookeeper' 'confluent-control-center' 'confluent-kafka-connect' 'confluent-kafka-rest' 'confluent-ksql' 'confluent-kafka-connect' 'confluent-schema-registry' 'mongod' 'samba' 'alfresco7')
```
Comment (bojmil-armedia) [inline vagrant/provisioning/roles/arkcase-developer/tasks/templates/start_services.sh:6]: Should you remove alfresco?

Comment (dzdimov) [reply vagrant/provisioning/roles/arkcase-developer/tasks/templates/start_services.sh:6]: alfresco should be removed or alfresco7. It depends when we are going to start with alfresco7 as default installation

Comment (bojmil-armedia) [reply vagrant/provisioning/roles/arkcase-developer/tasks/templates/start_services.sh:6]: This is for dev environments, so we should switch immediately. For production environments we should first test and then make it default.

File: vagrant/provisioning/roles/arkcase-app/tasks/main.yml [hunk 3]
```diff
@@ -76,6 +76,20 @@
     path: "{{ root_folder }}/common/symmetricKeyWithNewLine.decrypted"
     state: absent
 
+- name: remove pip2 that might come up along OS        
+  become: yes
```
Comment (david-oc-miller) [inline vagrant/provisioning/roles/arkcase-app/tasks/main.yml:80]: The name says "remove pip2" but the task removes pip3... the name and the task are not agreeing.

File: vagrant/provisioning/roles/arkcase-developer/tasks/templates/start_services.sh (hunk 2 above)
Comment (david-oc-miller) [reply vagrant/provisioning/roles/arkcase-developer/tasks/templates/start_services.sh:6]: Pretty sure Alfresco 7 became default with DSO-480.  At least in arkcase-ce.yml and arkcase-dev.yml (pretty sure) the alfresco tags call the alfresco7 roles.

**Pull request merged**